Kiosk/App/CardHandler.swift

```diff
@@ -1,9 +1,14 @@
 import UIKit
-import ReactiveCocoa
+import RxSwift
 
 class CardHandler: NSObject, CFTReaderDelegate {
 
-    let cardSwipedSignal = RACSubject()
+    private let _cardStatus = BehaviorSubject(value: "")
```

Comment:
Good question! I used it because I didn't know about the other one :P Thanks for the note, I'll change :bow: